test/cmdlineTests/evm_to_wasm_break/output

```diff
@@ -108,11 +108,18 @@ object "object" {
             default { z := 1:i32 }
             z4 := i64.extend_i32_u(z)
         }
-        function calldataload(x1, x2, x3, x4) -> z1, z2, z3, z4
+        function u256_to_i32(x1, x2, x3, x4) -> v:i32
         {
             if i64.ne(0, i64.or(i64.or(x1, x2), x3)) { unreachable() }
             if i64.ne(0, i64.shr_u(x4, 32)) { unreachable() }
-            eth.callDataCopy(0:i32, i32.wrap_i64(x4), 32:i32)
+            v := i32.wrap_i64(x4)
+        }
+        function calldataload(x1, x2, x3, x4) -> z1, z2, z3, z4
+        {
+            let _1 := 0
```

Comment:
Why do you need this variable?

Reply:
This is just the output of the compiler.

Reply:
Ah right, never mind!